name: "python_executor"
mode: ONCE
time_limit: 10
log_level: ERROR

clone_newuser: false
clone_newnet: false
keep_caps: false
keep_env: true
mount_proc: true

cwd: "/tmp"

mount { src: "/bin" dst: "/bin" is_bind: true }
mount { src: "/lib" dst: "/lib" is_bind: true }
mount { src: "/lib64" dst: "/lib64" is_bind: true mandatory: false }
mount { src: "/usr" dst: "/usr" is_bind: true }
mount { src: "/usr/local" dst: "/usr/local" is_bind: true }
mount { src: "/etc" dst: "/etc" is_bind: true }
mount { src: "/app" dst: "/app" is_bind: true rw: true }

mount {
  dst: "/tmp"
  fstype: "tmpfs"
  rw: true
  options: "size=512M"
}

mount { src: "/dev/null" dst: "/dev/null" is_bind: true rw: true }
mount { src: "/dev/urandom" dst: "/dev/urandom" is_bind: true }

exec_bin {
  path: "/usr/local/bin/python3"
  arg: "/tmp/user_script.py"
}
